sdk/storage/Microsoft.Azure.WebJobs.Extensions.Storage.Blobs/src/Listeners/SharedBlobQueueListener.cs

```diff
@@ -73,7 +73,7 @@ public void Dispose()
 
         public IScaleMonitor GetMonitor()
         {
-            return (IScaleMonitor)_listener;
+            return ((QueueListener)_listener).GetMonitor();
```

Comment:
`SharedBlobQueueListener `must also implement `ITargetScalerProvider` to get TBS for the blob trigegr.

Reply:
done